clr-k8s-examples/DEVELOP.md

```diff
@@ -1,6 +1,33 @@
 # Developer
 
-This document describes the key concepts and technologies used in the project.
+This document describes the key concepts and technologies used in the project and how to contribute.
```

Comment:
Just listing few Grammatical nits, feel free to edit as you like:
This document describes the key concepts and technologies used in the project, and lists the ways to contribute to the project.

Reply:
Fixed!